sentimental_analysis/realworld/views.py

```diff
@@ -232,18 +217,17 @@ def productanalysis(request):
                     stars = int(float(st))
                     if(stars != -1):
                         if(stars >= 4):
-                            r['pos'] += 0.1
+                            r['compound'] += 0.1
                         elif(stars >= 2):
-                            r['neu'] += 0.1
+                           continue
                         else:
-                            r['neg'] += 0.1
-                largest = 'pos'
-                if(r['neg'] > r[largest]):
-                    largest = 'neg'
-                if(r['neu'] > r[largest]):
-                    largest = 'neu'
-                reviews2[largest] += 1
-                
+                            r['compound'] -= 0.1
```

Comment:
Is the sum of 'r' always constant?
If not shouldn't we add some value relative to the sum of 'r' instead of '0.1'

Reply:
The value of r['compound'] ranges from -1 to +1, so we can add a small bias based on the starts given to the review. Hence adding or removing a 0.1 depending on the stars added to the review